content/more-on-functions/reading/pass-function-as-arguments/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Passing Functions as Arguments"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 3
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+*Functions are data*, and therefore can be passed around just like other values. This means a function can be *passed to another function* as an argument. This allows the *function being called* to use the *function argument* to carry out its action. This turns out to be extremely useful. 
```

Comment:
Please replace this with Functions are objects

Reply:
updated language